src/Features/LanguageServer/Protocol/Handler/CodeLens/CodeLensResolveHandler.cs

```diff
@@ -71,8 +71,8 @@ public async Task<LSP.CodeLens> HandleRequestAsync(LSP.CodeLens request, Request
                 CommandIdentifier = ClientReferencesCommand,
                 Arguments =
                 [
-                        resolveData.TextDocument.Uri,
-                        request.Range.Start
+                    resolveData.TextDocument.Uri,
+                    request.Range.Start
                 ]
```

Comment:
Do we not have an analyzer for these missing commas?

```suggestion
                    request.Range.Start,
                ],
```